browser/skus/skus_sdk_mojom_impl.cc

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) 2021 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at http://mozilla.org/MPL/2.0/.
+
+#include "brave/browser/skus/skus_sdk_mojom_impl.h"
+
+#include <memory>
+#include <utility>
+
+#include "brave/browser/skus/skus_sdk_service_factory.h"
+#include "brave/components/skus/browser/skus_sdk_service.h"
+#include "content/public/browser/browser_context.h"
+
+namespace brave_rewards {
+
+SkusSdkMojomImpl::SkusSdkMojomImpl(content::BrowserContext* context)
+    : service_(SkusSdkServiceFactory::GetForContext(context)) {}
+
+SkusSdkMojomImpl::~SkusSdkMojomImpl() {}
+
+void SkusSdkMojomImpl::RefreshOrder(const std::string& order_id,
+                                    RefreshOrderCallback callback) {
+  service_->RefreshOrder(order_id, std::move(callback));
```

Comment:
this service could be null - see https://github.com/brave/brave-core/pull/10847/files#r762255425

Reply:
Should be fixed now - implementation requires service in constructor
https://github.com/brave/brave-core/pull/10847/commits/780676bbebb2283387763229a4592c4eb4eadb9d

Reply:
Class was removed with https://github.com/brave/brave-core/commit/0c492c6a95c3b11eb0f4ab4c316510b78a030d19 /  https://github.com/brave/brave-core/pull/10847/commits/791f0b03062acdb751c210042bb3079b338ea051 / https://github.com/brave/brave-core/pull/10847/commits/2dacad8e9efaca7acf15e66420ba435fdc886644